src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -12,6 +12,8 @@
                 "switchToAdvancedMode": "Switch to advanced mode",
                 "advancedModeTooltip": "The advanced mode offers extended configuration options."
             },
+            "create": "Create Programming",
+            "import": "Import Programming",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys